logfile

```diff
@@ -0,0 +1,6 @@
+2017-12-11 20:42:08.278 CST [9476] FATAL:  database files are incompatible with server
```

Comment:
Shouldn't be committing this file so just delete it and push up again @CoderRx001